samza-core/src/main/java/org/apache/samza/runtime/LocalContainerRunner.java

```diff
@@ -50,18 +51,22 @@ public static void main(String[] args) throws Exception {
         }));
 
     String containerId = System.getenv(ShellCommandConfig.ENV_CONTAINER_ID);
-    log.info(String.format("Got container ID: %s", containerId));
     System.out.println(String.format("Container ID: %s", containerId));
 
     String coordinatorUrl = System.getenv(ShellCommandConfig.ENV_COORDINATOR_URL);
-    log.info(String.format("Got coordinator URL: %s", coordinatorUrl));
     System.out.println(String.format("Coordinator URL: %s", coordinatorUrl));
 
     Optional<String> execEnvContainerId = Optional.ofNullable(System.getenv(ShellCommandConfig.ENV_EXECUTION_ENV_CONTAINER_ID));
 
     int delay = new Random().nextInt(SamzaContainer.DEFAULT_READ_JOBMODEL_DELAY_MS()) + 1;
     JobModel jobModel = SamzaContainer.readJobModel(coordinatorUrl, delay);
     Config config = jobModel.getConfig();
+
+    // this call is also in LocalContainerRunner, but adding this here allows more logs to get handled by Samza loggers
```

Comment:
nit: was this meant to be "this call is also in ContainerLaunchUtil"

Reply:
Thanks!